### PR TITLE
Docs/aws access

### PR DESCRIPTION
**Pull request by dms1981**
Adds documentation to solve #1066 . Discusses how to access and assume role for privileged accounts. Adds some context into team tools used, and how these accounts are created.

### CODE REVIEW

File: source/team-guide/accessing-aws-accounts.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+We have two common methods for accessing AWS accounts:
+
+- Through a [common portal](https://moj.awsapps.com/start#/).
+- Through a [superuser account](https://user-guide.modernisation-platform.service.justice.gov.uk/team-guide/adding-a-new-team-member.html#adding-them-as-a-superadmin-in-our-aws-landing-zone).
+
+### Common portal access
+
+Using a web browser, an authenticated user can navigate to the [common portal](https://moj.awsapps.com/start#/) and select
+an AWS account. The portal will allow them to log in to the AWS console with read-only privileges, or to retrieve an
```
Comment (davidkelliott) [inline source/team-guide/accessing-aws-accounts.html.md.erb:19]:
```suggestion
an AWS account. The portal will allow them to log in to the AWS console with the relevant privileges, or to retrieve an
```
We have different levels, admin/developer/readonly/etc depending on the persons role and the account

**Review by davidkelliott — CHANGES_REQUESTED** [review]
Looks good thanks for doing this! One minor change to make please :)

**Review by davidkelliott — CHANGES_REQUESTED** [review]
Aha also could you add a link from the home page for the new page here please, cheers

https://github.com/ministryofjustice/modernisation-platform/blob/main/source/index.html.md.erb#L64

**Comment by dms1981** [comment]
> Aha also could you add a link from the home page for the new page here please, cheers
> 
> https://github.com/ministryofjustice/modernisation-platform/blob/main/source/index.html.md.erb#L64

Added in the most recent commit

**Review by davidkelliott — APPROVED** [review]

**Pull request merged**